rpm/SPECS/contextBroker.spec

```
@@ -72,7 +72,7 @@ fi
 
 %pre
 getent group %{owner} >/dev/null || groupadd -r %{owner}
-getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /opt/orion -s /bin/bash -c 'Orion account' %{owner}
+getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /home/orion/ -s /bin/bash -c 'Orion account' %{owner}
```

An additional improvement, pls:

```suggestion
getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /home/orion -s /bin/bash -c 'Orion account' %{owner}
```

---

Fixed in https://github.com/telefonicaid/fiware-orion/pull/3863/commits/c784068e40923126d867fbc253c8263abfca960f